news/2023/2023-08-12-new-featured-artist-kardashev.md

```diff
@@ -0,0 +1,55 @@
+---
+layout: post
+title: "New Featured Artist: Kardashev"
+date: 2023-08-12 13:00:00 +0000
+---
+
+Your fingers quiver as **Kardashev** engulfs our Featured Artist library.
+
+![](https://assets.ppy.sh/artists/375/header.jpeg)
+
+Merging death metal with shoegaze (and a heap of other flavours), **Kardashev** has been outputting a distinctly atmospheric style of metal for over a decade. And as of today, that sound is officially part of osu!.
+
+**12** mind-altering tracks can be found on [**Kardashev**'s Featured Artist listing](https://osu.ppy.sh/beatmaps/artists/375), including a few beatmap templates that combine certain intro/outro tracks, like [*The Approaching Atonement* + *Silvered Shadows*](https://osu.ppy.sh/beatmapsets/1889320). Feel free to start mapping any of these insanely dense songs (at your own risk).
+
+Speaking of hopeless cries for salvation, we asked [A L E P H](https://osu.ppy.sh/users/6735738) to play the infamous spaced-stream beatmap [*Cellar of Ghosts*](https://osu.ppy.sh/beatmapsets/1804362) at COE. See how that went for yourself:
+
+<div align="center">
+    <video width="85%" controls>
+        <source src="https://assets.ppy.sh/artists/375/release_showcase.mp4" type="video/mp4" preload="none">
+    </video>
+</div>
+
+### Kardashev - Cellar of Ghosts
+
+Attempt [the map from the video above](https://osu.ppy.sh/beatmapsets/1804362) by [morphne](https://osu.ppy.sh/users/12927633) and [iamtickrate](https://osu.ppy.sh/users/10772580)!
+
+<audio controls>
+    <source src="https://assets.ppy.sh/artists/375/Liminal%20Rite/Kardashev%20-%20Cellar%20of%20Ghosts.mp3">
+</audio>
+
+### Kardashev - Beside Cliffs and Chasms
+
+If you'd rather smash fingers on both of your hands, [this osu!taiko map](https://osu.ppy.sh/beatmapsets/1515274) by [Heaxys](https://osu.ppy.sh/users/5671417) was made for you.
+
+<audio controls>
+    <source src="https://assets.ppy.sh/artists/375/The%20Almanac/Kardashev%20-%20Beside%20Cliffs%20and%20Chasms.mp3">
+</audio>
+
+### Kardashev - Behind Leaves and Vines
+
+And here's [one more osu!taiko map](https://osu.ppy.sh/beatmapsets/1832856) also by [Heaxys](https://osu.ppy.sh/users/5671417)!
+
+<audio controls>
+    <source src="https://assets.ppy.sh/artists/375/The%20Almanac/Kardashev%20-%20Beside%20Cliffs%20and%20Chasms.mp3">
+</audio>
+
+---
+
+We have something to tell you. Come closer. You don't want to miss this. Okay, here it goes: ᵗʰᵉʳᵉ ᵃʳᵉ ᵐᵒʳᵉ **ᴷᵃʳᵈᵃˢʰᵉᵛ** ᵇᵉᵃᵗᵐᵃᵖ ᵗᵉᵐᵖˡᵃᵗᵉˢ [ʰᵉʳᵉ](https://osu.ppy.sh/beatmaps/artists/375)·
+
+The current batch of [**Featured Artist playlists** in osu!(lazer)](/wiki/People/Featured_Artists/Featured_Artist_playlists) close in **3 days**! Whoever's in the **top 10** by the end of August 15 earns progress towards a profile badge, so if you're looking for a shiny addition to your user page, this could be it.
```

Comment:
```suggestion
The current batch of [**Featured Artist playlists** in osu!(lazer)](/wiki/People/Featured_Artists/Featured_Artist_playlists) closes in **3 days**! Whoever's in the **top 10** by the end of August 15 earns progress towards a profile badge, so if you're looking for a shiny addition to your user page, this could be it.
```